phoenix-core/src/main/java/org/apache/phoenix/coprocessor/TaskRegionObserver.java

```diff
@@ -70,7 +70,11 @@
     private long timeMaxInterval = QueryServicesOptions.DEFAULT_TASK_HANDLING_MAX_INTERVAL_MS;
     @GuardedBy("TaskRegionObserver.class")
     // initial delay before the first task is handled
-    private static final long  initialDelay = 10000; // 10 secs
+    private static long initialDelay = 10000; // 10 secs
+
+    public static void disableTaskHandling() {
```

Comment:
We should make this method "package-private".

Reply:
If its package-private then we won't be able to call it from the BasePhoenixMetricsIT.java right? Generally we usually have a config option to set parameters such as these (for example see QueryServices.INDEX_REBUILD_TASK_INITIAL_DELAY). Then you would put it in the server properties and call setUpTestDriver(ReadOnlyProps serverProps, ReadOnlyProps clientProps)